_tutorials/2016-06-23-copying-telemetry-policy-files-in-ios-xr-6-0-1.md

```diff
@@ -44,3 +44,51 @@ sftp&gt; <mark>quit</mark>
 </code>
 </pre>
 </div>
+
+If you wish to copy a policy file from the host Linux operating system to the IOS XR filesystem within the Vagrant box, then there is a two-step process. Firstly, a writeable directory must be created in disk0:
+
+<div class="highlighter-rouge">
+<pre class="highlight">
+<code>
+RP/0/RP0/CPU0:ios#<mark>run</mark>
+Tue Jun 28 06:40:16.346 UTC
+[xr-vm_node0_RP0_CPU0:~]$<mark>cd /disk0\:</mark>
+[xr-vm_node0_RP0_CPU0:~]$<mark>mkdir mypolicies</mark>
+[xr-vm_node0_RP0_CPU0:~]$<mark>chmod 777 mypolicies</mark>
+</code>
+</pre>
+</div>
+
+The policy file can then be copied using scp to the Vagrant IOS XR instance:
+
+<div class="highlighter-rouge">
+<pre class="highlight">
+<code>
+user@host:~/vagrant/iosxrv> <mark>scp -P 2222 mypolicy.policy vagrant@localhost:/disk0:/mypolicies/</mark>
+vagrant@localhost's password: 
```

Comment:
I think @scadora 's original post is meant for any generic router setup. It would be more appropriate to submit something for a vagrant box as a separate tutorial and compare it. Also how did you determine port 2222? Probably using "vagrant port " command before you run the scp will make that clear.